rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -929,4 +956,33 @@ mod tests {
             expected
         );
     }
+
+    #[tokio::test]
+    async fn test_stop_points() {
```

Comment:
Would you mind including a test to ensure things work as expected in the case of an error?

Reply:
Added in ab3afdaa7c9b5faee97ba211f136e079324e2d25